docs/howto/distributed-tracing/proc_tracing-a-single-service.adoc

```diff
@@ -72,7 +72,7 @@ include::src/main/java/org/wildfly/swarm/howto/tracing/MySimpleResource.java[tag
 ----
 
 All JAX-RS resources are automatically traced.
-You can customize this behavior using the `@Traced` annotation from MicroProfile OpenTracing.
+You can customize this behavior using the `@Traced` annotation from Eclipse MicroProfile OpenTracing.
```

Comment:
This is part of the "Distributed Tracing" guide. Could we make sure the guide uses the full name somewhere at the beginning, and then leave the "Eclipse" word out? It reads worse now.